src/documentation/app/layout/sidebar/Sidebar.jsx

```diff
@@ -6,66 +6,73 @@ import { Link } from 'react-router-dom';
 import { ChevronDown, ChevronUp } from '@bufferapp/ui/Icon';
 import helper from 'immutability-helper';
 
-const SidebarWrapper = styled.div`
-    z-index: 15;
-    background: #F5F7F9;
-    display: flex;
-    flex-direction: column;
-    align-items: stretch;
-    width: 250px;
-    overflow-x: hidden;
-    height: calc(100vh - 65px);
-    padding: 0 0 0 1rem;
-    border-right: 1px solid #E6ECF1;
-    overflow-y: auto;
+const Section = styled.div`
+  z-index: 15;
+  background: #f5f7f9;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 250px;
+  overflow-x: hidden;
+  height: calc(100vh - 65px);
+  padding: 0 0 0 1rem;
+  border-right: 1px solid #e6ecf1;
+  overflow-y: auto;
 `;
```

Comment:
To make things consistent, do you think it might make sense to name this `SidebarSection` @gomezjuliana 😊 ?

Reply:
hmm wonder why this didn't change? This is the doc site UI's Sidebar and I changed it by mistake in a "replace all" spree 🙈 Got this changed back to <SidebarWrapper/>